extensions/wikia/Recirculation/js/recirculation.js

```diff
@@ -0,0 +1,44 @@
+/*global define*/
+define('ext.wikia.recirculation.recirculation', [
+	'wikia.log',
+	'wikia.window',
+	'wikia.abTest',
+	'videosmodule.controllers.rail'
+], function (log, win, abTest, videosModule) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.recirculation.recirculation',
+		$ = win.$;
+
+	function injectVideosModule ( element ) {
+		videosModule(element);
+	}
+
+	function injectTaboola( element ) {
+		win._taboola = win._taboola || [];
+		win._taboola.push({
```

Comment:
IMO this part should be moved to taboolaHelper . In fact the method could also load Taboola lib if needed. (We wouldn't need to expose the "loadTaboola" method then).